pkg/ifuzz/ifuzz.go

```diff
@@ -106,7 +106,9 @@ func randInsn(cfg *Config, r *rand.Rand) iset.Insn {
 		insns = insnset.GetInsns(cfg.Mode, iset.Type(r.Intn(3)))
 	} else if cfg.Priv {
 		insns = insnset.GetInsns(cfg.Mode, iset.Type(r.Intn(2)))
-	} else {
+	}
+	if len(insns) == 0 {
```

Comment:
Which set is empty? Is it Exec?
Perhaps it's better to add 1 stub pseudo-instruction, then we don't need any special cases.

Reply:
We could do that, but must a platform really have pseudo-instructions?

Reply:
Makes sense.

But it may also not have any TypeUser instructions :)